unstructured/errors.py

```diff
@@ -0,0 +1,4 @@
+class PdfMaxPagesExceededError(ValueError):
+    """Error raised, when number of PDF pages exceeds max_pages limit
```

Comment:
And if we make the above change, we can change the verbiage down here too to make it more generic.

Reply:
Additionally, @mackurzawa proposed to add some custom fields to this error class there but I stated it's enough to init with just a message.
Now I'm thinking if adding 2 fields:
- `document_pages: int`
- `max_pages: int`
would be helpful forming some custom messages in the downstream try-catch logic?

Reply:
Yeah I think that would be a good addition, since then the user knows how to split up the document to resolve this issue.